_commands/tools/ps.md

```diff
@@ -0,0 +1,80 @@
+---
+---
+
+ps
+-------
+
+'ps' provides a snapshot of current processes
```

Comment:
Looks good. Only suggestion is to please use inline code syntax with backticks (`) for snippets like `ps` and `ps -aux` across your documentation.